### ISSUE TITLE
Find a way to minimize the workload generated from invalid submitted reports

### ISSUE THREAD

**Issue by anadahz** [ooni/pipeline#27]
We should find a way to process non correctly formatted or invalid reports that are saturate the processing power of the pipeline.
If we decide to discard these reports we should do it in a smart way to counteract any potential collateral damage i.e, not processing reports from a whole country.

**Comment by darkk** [comment]
IMHO, that's very generic ticket that can never be solved in the pipeline as it's an ongoing process of incident response. So I'm closing it.

I think, incidents (like #139) should be resolved on case-by-case basis.

I think, if the collector has accepted the measurement from the user, than we have some "contract" to give it some "best-effort" service. That is perfectly summarized by "garbage in &mdash; garbage out" modality of the current pipeline code.

**Comment by anadahz** [comment]
I don't remember the exact incident I think that something/somebody was sending one report per URL from UK and this why (?) processing of AS2856, AS20712, AS5607 is ignored.
The relevant line in documentation: 
https://github.com/ooni/pipeline/blame/master/docs/readme-2.0.0.md#L205

Perhaps @hellais has more insights on this.
If it helps I can open a separate issue.

**Comment by darkk** [comment]
> processing of AS2856, AS20712, AS5607 is ignored

Those measurements are not ignored anymore and those measurements do not create significant issues by being treated as first-class dataset citizens.

Moreover, as far as I understand, MK will be sending one report per URL in near future. But @bassosimone knows better.

**Comment by hellais** [comment]
In the older model of the pipeline, we would start a new job for each report file (we were assuming there were not so many report files) and when some people started submitting 100s of thousands of report files the scheduled obviously got spammed and was not working properly.

As a temporary workaround to this we started ignoring those measurements while we figured how to refactor the scheduler logic to work on a bigger workload (more than 1 report file).

Now we have a very different logic in place where report files are all grouped together into big tarbals files and the scheduler works on data based on the canned filed, so this is no longer an issue.

I think it's reasonable to close this issue.
